sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -257,6 +259,14 @@ configKey
     : quotedIdentifier
     ;
 
+configValue
+    : quotedIdentifier
+    ;
+
+anything
```

Comment:
This is need for https://github.com/apache/spark/pull/30332/files#diff-52c70b37bde4ed4abb96887207dd6aa45a1785be1d31013203800c97592e580bR251

Reply:
`anything` -> `otherConfigKey`?

Reply:
or maybe we can replace `.*?` all over the file with `anything` in a follow-up?